tests/test_pois.py

```diff
@@ -0,0 +1,274 @@
+from flask import jsonify
+import json
+from datetime import date
+
+#Complete params
+
+poi_create_complete = {
+    'name': 'Morrow Plots',
+    'date': '2018-03-05',
+    'description': 'Just corn here',
+    'map_year': 1856,
+    'x_coord': 76,
+    'y_coord': 42,
+    'links': [
+        {
+            'link_url': 'http://fbnew.com',
+            'display_name': 'FacebookNew'
+        }
+    ],
+    'media': [
+        {
+            'content_url': 'http://images.com/alpaca.jpg',
+            'caption': 'The new alpaca in its natural habitat'
+        }
+    ],
+    'story_ids': [1, 2]
+}
+
+poi_update_complete = {
+    'name': 'Morrow Plots',
+    'date': '2018-03-05',
+    'description': 'Just corn here',
+    'links': [
+        {
+            'link_url': 'http://fbnew.com',
+            'display_name': 'FacebookNew'
+        }
+    ],
+    'media': [
+        {
+            'content_url': 'http://images.com/alpaca.jpg',
+            'caption': 'The new alpaca in its natural habitat'
+        }
+    ],
+    'story_ids': [1, 2]
+}
+
+#Permissible (only required attributes, optional ones not included)
+
+poi_create_permissible = {
+    #This one needs EITHER map_year OR story_ids
+    'name': 'Morrow Plots',
+    'date': '1856-01-01',
+    'description': 'Just corn here',
+    'map_year': 1856,
+    'x_coord': 76,
+    'y_coord': 42,
+    'links': [],
+    'media': [],
+    'story_ids': []
+}
+
+poi_edit_permissible = {
+    'name': 'Morrow Plots',
+    #Technically date is not required, but frontend will automatically send in date as 1/1/mapyear
+    #Otherwise there would be an error when running this test
+    'date': '1856-01-01',
+    'description': 'Just corn here',
+    'links': [],
+    'media': [],
+    'story_ids': []
+}
+
+#Invalid
+
+poi_create_invalid = {
+    #Does not have name, description, map_year, x_coord, y_coord
+    'name': '',
+    'date': '01-01-00',
+    'description': '',
+    # 'map_year': ,
+    # 'x_coord': ,
+    'y_coord': 42,
+    'links': [],
+    'media': [],
+    'story_ids': []
+}
+
+map_year_invalid = 0
+story_id_invalid = 0
+poi_id_invalid = 0
+
+os.environ['POI_ID'] = '1' #initialize POI_ID as environment variable
+
+class POITests(unittest.TestCase):
+
+    def test0_1_create_poi(self):
+        r = requests.post('http://127.0.0.1:5000/pois', json=poi_create_complete)
+        response = r.json()
+        self.assertEqual(response['code'], 201)
+        self.assertEqual(response['result']['poi']['name'], poi_create_complete['name'])
+        # self.assertEqual(response['result']['poi']['date'], poi_edit['date'])
+        self.assertEqual(response['result']['poi']['description'], poi_create_complete['description'])
+        self.assertEqual(response['result']['poi']['map_year'], poi_create_complete['map_year'])
+        self.assertEqual(response['result']['poi']['x_coord'], poi_create_complete['x_coord'])
+        self.assertEqual(response['result']['poi']['y_coord'], poi_create_complete['y_coord'])
```

Comment:
these are getting pretty repetitive. you can make this chunk more DRY by first setting `response['result']['poi']` as a variable and then perhaps looping through the attributes.